platform/darwin/docs/guides/Data-Driven Styling.md.ejs

```diff
@@ -33,48 +35,83 @@ The documentation for individual style properties will note which style function
-
-  let source = MGLShapeSource(identifier: "earthquakes", url: url, options: nil)
-  style.addSource(source)
-
-  let stops = [0: MGLStyleValue(rawValue: <%- cocoaPrefix %>Color.yellow),
-               2.5: MGLStyleValue(rawValue: <%- cocoaPrefix %>Color.orange),
-               5: MGLStyleValue(rawValue: <%- cocoaPrefix %>Color.red),
-               7.5: MGLStyleValue(rawValue: <%- cocoaPrefix %>Color.blue),
-               10: MGLStyleValue(rawValue: <%- cocoaPrefix %>Color.white)]
-
-  let layer = MGLCircleStyleLayer(identifier: "circles", source: source)
-  layer.circleColor = MGLStyleValue(interpolationMode: .exponential,
-                                    sourceStops: stops,
-                                    attributeName: "mag",
-                                    options: [.defaultValue: MGLStyleValue<<%- cocoaPrefix %>Color>(rawValue: .green)])
-  layer.circleRadius = MGLStyleValue(rawValue: 10)
-  style.insertLayer(layer, below: symbolLayer)
+let url = URL(string: "https://earthquake.usgs.gov/earthquakes/feed/v1.0/summary/all_week.geojson")
+let symbolSource = MGLSource(identifier: "source")
+let symbolLayer = MGLSymbolStyleLayer(identifier: "place-city-sm", source: symbolSource)
+
+let source = MGLShapeSource(identifier: "earthquakes", url: url, options: nil)
+style.addSource(source)
+
+let stops = [0: MGLStyleValue(rawValue: <%- cocoaPrefix %>Color.yellow),
+             2.5: MGLStyleValue(rawValue: <%- cocoaPrefix %>Color.orange),
+             5: MGLStyleValue(rawValue: <%- cocoaPrefix %>Color.red),
+             7.5: MGLStyleValue(rawValue: <%- cocoaPrefix %>Color.blue),
+             10: MGLStyleValue(rawValue: <%- cocoaPrefix %>Color.white)]
+
+let layer = MGLCircleStyleLayer(identifier: "circles", source: source)
+layer.circleColor = MGLStyleValue(interpolationMode: .exponential,
+                                  sourceStops: stops,
+                                  attributeName: "mag",
+                                  options: [.defaultValue: MGLStyleValue<<%- cocoaPrefix %>Color>(rawValue: .green)])
+layer.circleRadius = MGLStyleValue(rawValue: 10)
+style.insertLayer(layer, below: symbolLayer)
 ```
 
 ![exponential mode](img/data-driven-styling/exponential.png)
 
+### Exponential
+
+`MGLInterpolationModelExponential` combined with any `MGLStyleFunctionOptionInterpolationBase` greater than `0`, you can interpolate between values exponentially, create an accelerated ramp effect.
+
+Here's a visualization from the Mapbox Studio comparing interpolation base values of `1.5` and `0.5` based on zoom (see [Working with Mapbox Studio](working-with-mapbox-studio.html)).
+
+<img src="img/data-driven-styling/exponential-function.png" height=344/>
+<img src="img/data-driven-styling/exponential-function-1.png" height=344/>
+
+The example below illustrates a layer's `circleRadius` increasing exponentially by a map's zoom level with an `MGLStyleFunctionOptionInterpolationBase` of `1.5`.
```

Comment:
This sentence feels like it could use more commas or breaks, to clarify boundaries between complex ideas/words.